applications/app/controllers/WebAppController.scala

```diff
@@ -50,10 +50,11 @@ object WebAppController extends Controller with ExecutionContexts with Logging {
           Static("stylesheets/head.content.css"),
           Static("stylesheets/content.css"),
           Static("stylesheets/print.css"),
-          Static("javascripts/core.js"),
-          Static("javascripts/bootstraps/standard.js"),
+          Static("javascripts/curl+boostraps/standard+boot.js"),
```

Comment:
I think we could do a better job with filenames to avoid ambiguity. `bootstraps/standard` is the name of the AMD module, maybe we could just call this `standard`?